docs/ddl-of-carbondata.md

```diff
@@ -793,6 +793,19 @@ Users can specify which columns to include and exclude for local dictionary gene
        ALTER TABLE tablename UNSET TBLPROPERTIES('SORT_SCOPE')
        ```
 
+     - ##### SORT COLUMNS
+       Example to SET SORT COLUMNS:
```

Comment:
It will be helpful if we can provide a small description about the sort columns, for example:
1. What does this do?
2. In what scenarios user need to use this
3. Is there any advantages?
4. Is there any limitations?

Reply:
in "CREATE TABLE" section, it already provided these information.

Reply:
How about adding some description for below 2 points:
In what scenarios user need to use this ( ex: the table can be sorted after the creation of the table by using the following....)
Is there any advantages?

Reply:
ok